File: Website/App_Code/CConfig.vb
Imports Microsoft.VisualBasic

Public Class CConfig : Inherits CConfigBase

    Public Shared Function Layout() As ELayout
        Dim s As String = Config("Layout", "Horizontal")
        Return CType([Enum].Parse(GetType(ELayout), s), ELayout)
    End Function
    Public Shared Function MenuTitle() As String
        Return Config("MenuTitle", "Picasso")
    End Function

    Public Shared Function IsDev() As Boolean
        Return HttpContext.Current.Request.Url.Host.ToLower.Contains("localhost")
    End Function
End Class

File: WebsiteCSharp/App_Code/CConfig.cs

using System;
using System.Web;

public class CConfig : Framework.CConfigBase
{

    public static ELayout Layout()
    {
        string s = Config("Layout", "Horizontal");
        return ((ELayout)(Enum.Parse(typeof(ELayout), s)));
    }

    public static string MenuTitle()
    {
        return Config("MenuTitle", "Picasso");
    }

    public static bool IsDev()
    {
        return HttpContext.Current.Request.Url.Host.ToLower().Contains("localhost");
    }
}